EIPS/eip-tbd.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
```suggestion
---
eip: 8030
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.